pandas/tests/plotting/test_frame.py

```diff
@@ -1030,6 +1030,43 @@ def test_plot_scatter(self):
         axes = df.plot(x='x', y='y', kind='scatter', subplots=True)
         self._check_axes_shape(axes, axes_num=1, layout=(1, 1))
 
+    @pytest.mark.slow
+    def test_if_scatterplot_colorbar_affects_xaxis_visibility(self):
+        random_array = np.random.random((1000,3))
```

Comment:
PEP8.

You can run `flake8 pandas/tests/plotting/test_frame.py` locally to check this.